csharp-basics/exercises/Arithmetic/Exercise 11/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+public class Program
+{
+    public static string Moran(int num)
+    {
+        int rem = 0;
+        int sum = 0;
+        int n = num;
+
+        while (num > 0)
+        {
+            rem = num % 10;
+            sum = sum + rem;
+            num = num / 10;
+        }
+        bool isMoran = false;
+        if (n % sum == 0)
+        {
+            int quotient = n / sum;
+            for (int i = 2; i <= Math.Sqrt(quotient); i++)
+            {
+                if (quotient % i == 0)
+                {
+                    isMoran = false;
+                    break;
+                }
+                else
+                {
+                    isMoran = true;
+                }
+            }
```

Comment:
Šajā uzd ir gūti saprast, kas te notiek. Ir šis isMoran = true; un pēc tam vēl seko if (isMoran)
        {
            return "M";
        }